modules/cava/hm.nix

```
@@ -0,0 +1,29 @@
+{ config, lib, ... }:
+
+let 
+
+  mkGradient = colors: lib.listToAttrs (lib.imap0 (i: c: lib.nameValuePair "gradient_color_${toString (i+1)}" "'#${c}'") colors) // {
+    gradient = "1";
+    gradient_count = toString (builtins.length colors);
```

If the Home Manager modules implicitly converts these values to strings, consider using:

```suggestion
    gradient = 1;
    gradient_count = builtins.length colors;
```

Otherwise:

```suggestion
    gradient = toString 1;
    gradient_count = toString (builtins.length colors);
```